staff/acct/note

```diff
@@ -34,6 +34,10 @@ def guess_topic(note):
 
 
 def main():
+    if getpass.getuser() == 'root':
+        print('Do not run this script as root. Aborting.')
+        return 6
```

Comment:
`6`?

Reply:
3,4,5 was taken already in this script. no significant reason behind it.